### PR TITLE
AMW-215 Fixing linting issues

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/AMW-215

### CODE REVIEW

File: molecule/requirements.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ collections:
   - name: redhat.rhbk
   - name: redhat.data_grid
     version: '>=1.1.4'
+  - name: redhat.sso
```
Comment (guidograzioli) [inline molecule/requirements.yml]: redhat.sso is not needed

File: requirements.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 collections:
-  - name: redhat.data_grid
   - name: redhat.rhbk
+  - name: redhat.data_grid
+    version: '>=1.1.4'
+  - name: redhat.sso
```
Comment (guidograzioli) [inline requirements.yml]: also not needed here

**Review by guidograzioli — CHANGES_REQUESTED** [review]
remove redhat.sso from deps; otherwise LGTM

File: requirements.yml (hunk 2 above)
Comment (RanabirChakraborty) [reply requirements.yml]: Thanks! I didn't see you have removed the sso playbook, removed it now. review once.

**Review by guidograzioli — APPROVED** [review]

**Pull request merged**